### ISSUE TITLE
Representing differentials in Expr outside of Derivatives/Integrals

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#1407]
The derivation for moment equilibrium I wrote in the case studies repo includes a differential outside of the context of a derivative or integral:

![image](https://user-images.githubusercontent.com/18634568/58350511-1491a800-7e34-11e9-9a1d-028e8c955161.png)

There doesn't seem to be a way currently to represent a standalone differential in `Expr` (differentials only show up within derivatives and integrals in Expr). I could probably rephrase this to not include the standalone differential, since it does end up inside an integral later on, but differentials seem like something that should be able to exist on their own in `Expr`.

**Comment by JacquesCarette** [comment]
Agreed - but hopefully this isn't too high priority?

**Comment by bmaclach** [comment]
I don't think so. I'm planning on just leaving a FIXME in the derivation for now.

**Comment by smiths** [comment]
Yes, let's leave a fixme for now.
